### PR TITLE
feat(workspace): SD-3411 By using keyboard shortcuts, change the current showed view

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
Main task: SD-774

### CODE REVIEW

File: client/app/scripts/superdesk-workspace/workspace.js [hunk 1]
```diff
@@ -324,6 +324,36 @@
                         superdeskFlags.flags.hideMonitoring = false;
                     }
                 };
+
+                /*
+                 * By using keyboard shortcuts, change the current showed view
+                 *
+                 */
+                var opt = {global: true};
+                keyboardManager.bind('alt+h', function (e) {
```
Comment (petrjasek) [inline client/app/scripts/superdesk-workspace/workspace.js]: we were using `ctrl` + something shortcuts, so not sure we should use `alt` for these, what do you think @fritzSF ?

Comment (pavlovicnemanja) [inline client/app/scripts/superdesk-workspace/workspace.js]: I had a talk with @fritzSF about this this morning, and we agreed to use `alt +` for global shortcuts (like changing views) and `ctrl +` for authoring shortcuts.

Comment (petrjasek) [inline client/app/scripts/superdesk-workspace/workspace.js]: ok then

**Comment by petrjasek** [comment]
does it unbind when you get out of workspace or it works always?

**Comment by pavlovicnemanja** [comment]
No, it stays... Wherever you are, when you press some combination, url changes.

**Comment by petrjasek** [comment]
ok that should be fine
:+1:

**Comment by plamut** [comment]
Do we test keyboard shortcuts?

Looks good though. :+1:

**Comment by pavlovicnemanja** [comment]
Yeah, I need to write e2e tests for them! :-) I will update this PR today ;-)

**Comment by plamut** [comment]
OK, good, I added the WIP label so that this does not get merged prematurely. :)

**Pull request merged**